header/header.html

```diff
@@ -3,12 +3,12 @@
         <div class="col-md-2 col-1"></div>
         <div class="col">
             <nav id="navbar" class="navbar">
-                <a class="navbar-brand" href="#">
+                <a class="navbar-brand" href="index.html">
                     <img id="brand_img" src="assets/worktezName.png" alt="Worktez" width="125px" height="45px">
                 </a>
                 <ul class="nav nav-pills" style="display: inline-flex; font-size: 12px;">
                     <li class="nav-item" >
-                        <a class="nav-link" href="pricing/pricing" style="color: #838CAB;">Pricing</a>
+                        <a class="nav-link" href="/pricing/pricing.html" style="color: #838CAB;">Pricing</a>
                     </li>
```

Comment:
The .html was purposely removed to remove the .HTML extension in the URL.
Please follow the same.